pkgs/kde/gear/kio-extras/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  mkKdeDerivation,
+  qt5compat,
+  qtsvg,
+  pkg-config,
+  libkexiv2,
+  samba,
+  libssh,
+  libmtp,
+  libimobiledevice,
+  gperf,
+  libtirpc,
+  openexr,
+  taglib,
+  shared-mime-info,
+  libappimage,
+}:
+mkKdeDerivation {
+  pname = "kio-extras";
```

Comment:
I believe nixpkgs#155405 fix is also needed here; mtp access is not working for me

Reply:
Thanks, applied.

Reply:
Thank you, working fine now